website/src/content/blog/2025-03-31-typespec-1-0-release/type_spec_1_0_release.md

```diff
@@ -0,0 +1,202 @@
+
+For example, a generated JavaScript library will allow you to interact with your `todo` API like this:
+
+```javascript
+const client = new TodoServiceClient();
+const todo = await client.createTodoItem({
+  content: "Buy groceries",
+  dueDate: "2025-03-21",
+  isCompleted: false,
+});
+console.log(todo);
+```
+
+TypeSpec simplifies maintenance by allowing seamless updates to server and client code as APIs evolve, enabling you to focus on business logic rather than repetitive tasks.
+
+## Emitters: From TypeSpec to the API format you need
+
+TypeSpec's core value proposition is the ability to define your API once and generate multiple output formats. The 1.0-RC release ships with a stable OpenAPI 3.0 emitter (`@typespec/openapi3`), enabling you to seamlessly integrate TypeSpec into existing OpenAPI workflows and toolchains.
+
+But our emitter story doesn't stop there. The TypeSpec ecosystem includes several additional emitters:
+
+- **OpenAPI 2.0 Emitter**: For teams that still require Swagger/OpenAPI 2.0 compatibility
+- **REST Emitter**: Produces JSON descriptions of REST APIs
+- **Protocol Buffer Emitter**: Generates Protocol Buffer (protobuf) definitions
+- **JsonSchema Emitter**: Creates JSON Schema documents from your TypeSpec types
+
+While some of these emitters remain in preview as we gather feedback, they demonstrate the extensibility of TypeSpec across different API description formats.
+
+For teams with existing OpenAPI definitions looking to adopt TypeSpec, we've also created the OpenAPI to TypeSpec converter tool. This tool helps you migrate your existing API definitions to TypeSpec, providing a smoother onboarding experience and preserving your investment in OpenAPI. You can find the converter in our [GitHub repository](https://github.com/microsoft/typespec/tree/main/packages/openapi3-to-typespec).
+
+For teams with specialized needs, TypeSpec 1.0-RC includes a new [emitter framework](https://github.com/alloy-framework/alloy) that allows you to build custom emitters. This framework provides access to TypeSpec's compiler APIs and type system, enabling you to transform your TypeSpec definitions into any format your organization requires.
+
+We'll be stabilizing both additional emitters and the emitter framework based on your feedback in upcoming releases. If you're interested in creating custom emitters, check out our [documentation](https://typespec.io/docs/emitters/overview) for guidance and examples.
+
+## Stability of core components
+
+The core components of TypeSpec, including the compiler, HTTP library, OpenAPI emitters, and VS/VSCode extensions, are in a stable state and ready for production use. These components have been thoroughly tested and are designed to provide a reliable foundation for your API development workflows.
```

Comment:
Release candidates are not stable releases and are not recommended for production use without careful consideration. We are not promising we won't break anything, though the bar is extremely high.